templates/ReactReduxSpa/ClientApp/components/FetchData.tsx

```diff
@@ -56,8 +56,8 @@ class FetchData extends React.Component<WeatherForecastProps, {}> {
     }
 
     private renderPagination() {
-        let prevStartDateIndex = this.props.startDateIndex - 5;
-        let nextStartDateIndex = this.props.startDateIndex + 5;
+        let prevStartDateIndex = (this.props.startDateIndex || 0) - 5;
```

Comment:
Usually `startDateIndex` gets set in `componentWillMount` by calling `requestWeatherForecasts`.